lib/reporter.js

```diff
@@ -177,11 +178,12 @@ export default class Reporter {
 
   addPackageMetadata (error) {
     let activePackages = atom.packages.getActivePackages()
+    const availablePackagePaths = atom.packages.getPackageDirPaths()
     if (activePackages.length > 0) {
       let userPackages = {}
       let bundledPackages = {}
       for (let pack of atom.packages.getActivePackages()) {
-        if (/\/app\.asar\//.test(pack.path)) {
+        if (atom.packages.isBundledPackage(pack.name) && !availablePackagePaths.includes(path.dirname(pack.path))) {
```

Comment:
Why did you need to include `isBundledPackage` again for specs?  Shouldn't they still pass if you only spy on `getPackageDirPaths`?

Reply:
@50Wliu :+1: I changed this in https://github.com/atom/exception-reporting/pull/37/commits/5cd0fe788512f489c6c7608f53feec2d46ed1d44